server.js

```diff
@@ -3,21 +3,85 @@ const cors = require("cors");
 
 const app = express();
 
-app.use(cors());
+const bodyParser = require("body-parser");
+app.use(bodyParser.urlencoded({ extended: true }));
 
-const welcomeMessage = {
+app.use(cors());
+app.use(express.json());
+const welcomeMessage = 
+{
   id: 0,
   from: "Bart",
   text: "Welcome to CYF chat system!",
-};
+}
+
+  
 
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
+// get html page
+app.get("/", function (req, res) {
+  res.sendFile(__dirname + "/index.html");
+});
+// get all messages
+app.get("/messages", function(req, res){
+  res.status(200).send({messages});
+
+})
+//post new messages
+
+app.post("/messages", function(req, res){
+  console.log(req.body);
```

Comment:
Remember to remove console.log before committing. Bookmark to show you how to debug in VSCode so you have another way to check what your code is doing.